src/tools/dart-compile.md

```diff
@@ -74,39 +75,43 @@ The following table shows the subcommands of `dart compile`.
   <tr>
     <td> <code>exe</code> </td>
     <td> <span style="white-space: nowrap">Self-contained</span> executable </td>
-    <td> A standalone, architecture-specific executable file.
-      <a href="#exe">Learn more.</a>
+    <td> A standalone, architecture-specific executable file containing the source code
+      compiled to machine code and a small <a href="/overview#runtime">Dart runtime</a>.
+      <br><i><a href="#exe">Learn more.</a></i>
```

Comment:
```suggestion
      <br><em><a href="#exe">Learn more.</a></em>
```

Reply:
It's like @mit-mit is writing HTML from...1997 😜

Reply:
I enrolled in my C.S. program in 1994, so probably more like 1995/1996 ;-)